src/driver/amdxdna/amdxdna_drv.c

```diff
@@ -294,6 +304,10 @@ static int amdxdna_probe(struct pci_dev *pdev, const struct pci_device_id *id)
 		goto failed_dev_fini;
 	}
 
+	pm_runtime_set_autosuspend_delay(dev, 5000);
```

Comment:
```suggestion
	pm_runtime_set_autosuspend_delay(dev, 5000 /* ms */);
```
or define a macro for better readability.